lib/actions/oauth_signup_action.rb

```diff
@@ -63,13 +63,11 @@ def determine_name(auth)
     name = ''
```

Comment:
Is this line still needed if we're taking out the rest of the definition?

Reply:
Surprised the linter allows that unused variable. Must be used somewhere? If so, maybe its important to set it?

Reply:
__
```suggestion
```